### PR TITLE
Enable building and publishing of binaries via flowzone

### PR DESCRIPTION
**Pull request by rahul-thakoor**
Change-type: patch

### CODE REVIEW

File: .github/workflows/flowzone.yml [hunk 1]
```diff
@@ -1,20 +1,29 @@
-name: Flowzone
```
Comment (klutchell) [inline .github/workflows/flowzone.yml:1]: why did you remove this?

File: Cross.toml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    "apt-get update && apt-get install --assume-yes libdbus-1-dev:$CROSS_DEB_ARCH",
 ]
 
 [target.i686-unknown-linux-gnu]
 pre-build = [
     "dpkg --add-architecture $CROSS_DEB_ARCH",
-    "apt-get update && apt-get install --assume-yes libdbus-1-dev:$CROSS_DEB_ARCH"
-]
+    "apt-get update && apt-get install --assume-yes libdbus-1-dev:$CROSS_DEB_ARCH",
+]
```
Comment (klutchell) [inline Cross.toml:23]: Did anything change in this file besides formatting? I would prefer we not commit this file if functionally unchanged

File: .github/workflows/flowzone.yml (hunk 1 above)
Comment (rahul-thakoor) [reply .github/workflows/flowzone.yml:1]: i simply replicated the one we have at https://github.com/balena-os/takeover/blob/master/.github/workflows/flowzone.yml

File: Cross.toml (hunk 2 above)
Comment (rahul-thakoor) [reply Cross.toml:23]: only formatting. will leave it out

File: .github/workflows/flowzone.yml (hunk 1 above)
Comment (klutchell) [reply .github/workflows/flowzone.yml:1]: keep the name, it looks better in Action runs

Comment (rahul-thakoor) [reply .github/workflows/flowzone.yml:1]: it's specified here https://github.com/balena-os/wifi-connect/blob/dda3275fe7b7cac68350c64d74d97c464af17c0a/.github/workflows/flowzone.yml#L12

Comment (rahul-thakoor) [reply .github/workflows/flowzone.yml:1]: which is better? at the root or at the job level @klutchell ?

Comment (klutchell) [reply .github/workflows/flowzone.yml:1]: Both will show up, look at the status checks below...

Do you prefer

`.github/workflows/flowzone.yml / Flowzone / Versioned source (pull_request)`

or `Flowzone / Flowzone / Versioned source (pull_request)`

The latter looks better to me, and it's what you get when you add a name to the top level of the workflow.

Comment (rahul-thakoor) [reply .github/workflows/flowzone.yml:1]: I see, TIL! thanks

**Review by klutchell — APPROVED** [review]
